Tests/Microsoft.AppCenter.Test.Functional/Analytics/TrackEventTest.cs

```diff
@@ -10,10 +10,23 @@
 namespace Microsoft.AppCenter.Test.Functional.Analytics
 {
     using Analytics = Microsoft.AppCenter.Analytics.Analytics;
+    using AppCenter = Microsoft.AppCenter.AppCenter;
 
     public class TrackEventTest
     {
-        private readonly string _appSecret = Guid.NewGuid().ToString();
+
+        // Before
+        public TrackEventTest()
+        {
+            Utils.deleteDatabase();
+        }
+
+        // After
+        public void Dispose()
+        {
+            // Let pending SDK calls be completed.
+            Task.Delay(3000).Wait();
```

Comment:
The test already waits since it verifies what was sent so thats blocking for 3s until the call is done. We should try to remove this.

Reply:
Will be fixed separately